docs/reference/content/apidocs.md

```diff
@@ -0,0 +1,21 @@
++++
+date = "2020-01-28T15:50:55Z"
+title = "API Documentation"
+[menu.main]
+  name = "API Documentation"
+  pre = "<i class='fa fa-file-text-o'></i>"
+  weight = 90
+  identifier = "apiDocs"
++++
+
+## API Documentation
+
+The following API documentation is available:
+
+  - [Bson]({{< apiref "bson" "index.html" >}})
```

Comment:
I'm finding this confusing as a single list with the names as is.  How about something like this instead:

**Drivers**

- Java Driver (new API) 
- Java Driver (legacy API)
- Java Reactive Streams Driver
- Scala Driver

**Common**

- Common Driver API (common API shared by all of the above drivers)
- BSON 

Maybe we don't even need a direct link to mongodb-driver-core.  Is it enough to get there indirectly through the drivers?

Reply:
Just used a single list without common

Reply:
Looks good!